src/BUILD

```diff
@@ -864,10 +859,14 @@ cc_library( # make ovms_lib dependent, use share doptions
     srcs = [
             "s3filesystem.cpp",
     ],
-    deps = [
-        "@awssdk//:s3",
-        "@awssdk//:core",
-        "@awssdk//:deps",
+    deps = select({
+            "//:ubuntu_build": [
+                "@aws-sdk-cpp//:aws-sdk-cpp_ubuntu",
+            ],
+            "//:redhat_build": [
+                "@aws-sdk-cpp//:aws-sdk-cpp_redhat",
```

Comment:
how about using dashes in whole target name?

Reply:
IMO it actually distinguish target from some separate gimicks for ubuntu/redhat.